src/pages/HomePage.js

```diff
@@ -34,55 +34,61 @@ const HomePage = () => {
-                    </div>
-                    <div className='py-4'>
-                        <div className='text-3xl font-bold'>What is your cause?</div>
-                        
-                        <div className='text-xl pl-4'>Enter your fundraiser information</div>
-                    </div>
-                    <div className='py-4'>
-                        <div className='text-3xl font-bold'>Support</div>
-                        
-                        <div className='text-xl pl-4'>Enter your payment information</div>
-                    </div>
-                </div>
-
-            </div>
             
             {/* FEATURED FUNDRAISER */}
-            <div className='pt-16 px-8' data-testid="feat-fundraiser-test-id">
+            <div className='px-8 py-4' data-testid="feat-fundraiser-test-id">
                 {/* THe list of objects from the post request is being set as the props */}
                 <HomePageFeatFundraiser data={data}/>
             </div>
+
             {/* CARROUSSEL FUNDRAISER SLIDES */}
-            <div className='pt-16' data-testid="carousel-test-id">
-                <div className='pl-8 text-5xl font-HomePageHeading font-semibold'>
-                    General Fundraisers
+            <div className='px-8 py-4' data-testid="carousel-test-id">
+                <div className='text-5xl font-HomePageHeading font-semibold'>
+                    Active Fundraisers
                 </div>
                 <FundraiserCarousel data={data}/>
             </div>
+
+
+            {/* STEPS TO FUNDRAISER */}
+            <div className='px-8 pt-4 pb-8'>
+                <p className='pb-4 text-5xl font-HomePageHeading font-semibold'>
+                  Start your own Fundraiser
+                </p>
+                <div className='flex'>
+                    <img src={StepsPic} alt="Logo" className='w-1/3 h-1/3' />
+                    <div className='flex flex-col justify-center mx-8 space-y-8'>
+                        
+                        <div className=''>
+                            <div className='text-3xl font-bold'>The Basics</div>
+                            
+                            <div className='text-xl pl-4'>Enter your information</div>
+                        </div>
+                        <div className=''>
+                            <div className='text-3xl font-bold'>What is your cause?</div>
+                            
+                            <div className='text-xl pl-4'>Enter your fundraiser information</div>
+                        </div>
+                        <div className=''>
+                            <div className='text-3xl font-bold'>Support</div>
+                            
+                            <div className='text-xl pl-4'>Enter your payment information</div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            
```

Comment:
You mentioned on discord that we could use a website wide footer, could you make one?

Reply:
If you guys think that's a good idea, I'm sure I can.   I think it would be a good addition, just my opinion.

Reply:
Hell yeah!

Reply:
we could totally use one. For inspiration on website stuff ive been looking at gofundme and kickstarter